e2e/pages/events/addStatementOfServiceEvent.page.js

```diff
@@ -35,9 +30,8 @@ module.exports = {
     I.click(this.fields(elementIndex).recipients.addressCheck);
     this.enterRecipientsAddress(elementIndex, recipients);
     I.fillField(this.fields(elementIndex).recipients.documents, recipients.documents);
-    I.fillField(this.fields(elementIndex).recipients.date.day, recipients.date.day);
-    I.fillField(this.fields(elementIndex).recipients.date.month, recipients.date.month);
-    I.fillField(this.fields(elementIndex).recipients.date.year, recipients.date.year);
+    const dateSelector = `(//*[contains(@class, "collection-title")])[${ elementIndex + 1 }]/parent::div//*[contains(@class,"form-date")]`;
```

Comment:
again, is it worth overwriting the `this.fields(elementIndex).recipients.date ` value with this new xpath? to follow the existing pattern?